packages/berlin/src/pages/Cycle.tsx

```diff
@@ -137,11 +142,37 @@ function Cycle() {
   };
 
   const sortedOptions = useMemo(() => {
-    const sorted = [...(currentCycle?.questionOptions ?? [])].sort(
-      (a, b) => b.voteScore - a.voteScore,
-    );
+    const { column, order } = sorting;
+    const sorted = [...(currentCycle?.questionOptions ?? [])].sort((a, b) => {
+      if (column === 'author') {
+        const usernameA = a.user.username.toUpperCase();
+        const usernameB = b.user.username.toUpperCase();
+        return order === 'desc'
+          ? usernameB.localeCompare(usernameA)
+          : usernameA.localeCompare(usernameB);
+      } else if (column === 'affiliation') {
+        const affiliationA = a.user.group.name.toUpperCase();
+        const affiliationB = b.user.group.name.toUpperCase();
+        return order === 'desc'
+          ? affiliationB.localeCompare(affiliationA)
+          : affiliationA.localeCompare(affiliationB);
+      } else if (column === 'numOfVotes') {
+        const votesA = localUserVotes.find((vote) => vote.optionId === a.id)?.numOfVotes || 0;
+        const votesB = localUserVotes.find((vote) => vote.optionId === b.id)?.numOfVotes || 0;
+        return order === 'desc' ? votesB - votesA : votesA - votesB;
+      } else {
+        return order === 'desc' ? b.voteScore - a.voteScore : a.voteScore - b.voteScore;
+      }
```

Comment:
extract this to 3 smaller functions:

sortAuthor(a: string, b: string): number
sortVotes
sortNumOfVotes